packages/ia-zendesk-help-widget/src/styles/zendesk-widget.js

```diff
@@ -0,0 +1,43 @@
+import { css } from 'lit-element';
+
+export default css`
```

Comment:
_Suggestion_ You can put your style directly inside your WebComponent so you don't have to jump between files to see what your component is doing.

Reply:
I just wanted to make in its own file so that things can be managed for future enhancements.

Reply:
How does this help manage future enhancements?

Reply:
The main component file looks optimised because we pulled CSS in separate file. and later if we want to make more changes and want to add more CSS, that will consume more space in main file.

Reply:
If I understood you correctly, we want to make changes something like 
`static get styles() {
    return css`
      :host {
        --buttonBlue: #194880;
        --white: #fff;

        --iconFillColor: var(--white);
        --linkColor: var(--white);
      }`

Reply:
I will change something like this so that execute won't switch over files to get styling for this.